addon-test-support/properties/collection/legacy.js

```diff
@@ -5,7 +5,7 @@ import { create } from '../../create';
 import { count } from '../count';
 import Ceibo from 'ceibo';
 
-const arrayDelegateMethods = ['map', 'filter', 'mapBy', 'filterBy', 'forEach'];
+const arrayDelegateMethods = ['map', 'filter', 'mapBy', 'filterBy', 'findOneBy', 'findOne', 'forEach'];
```

Comment:
I suspect it does not not work for legacy collection cause it delegated to non-existing methods of the native Array? Maybe we should just take legacy collection alone?